File: game/src/main/java/com/bartlomiejpluta/base/game/image/asset/ImageAsset.java
package com.bartlomiejpluta.base.game.image.asset;

import com.bartlomiejpluta.base.game.common.asset.Asset;
import lombok.NonNull;

public class ImageAsset extends Asset {
   public ImageAsset(@NonNull String uid, @NonNull String source) {
      super(uid, source);
   }
}

File: editor/src/main/kotlin/com/bartlomiejpluta/base/editor/image/asset/ImageAsset.kt
package com.bartlomiejpluta.base.editor.image.asset

import com.bartlomiejpluta.base.editor.asset.model.GraphicAsset
import com.bartlomiejpluta.base.editor.project.model.Project

class ImageAsset(project: Project, uid: String, source: String, name: String) :
   GraphicAsset(project.imagesDirectoryProperty, uid, source, name)